docs/source/03_tutorial/07_set_up_experiment_tracking.md

```diff
@@ -0,0 +1,157 @@
+```
+
+The next step is to ensure that the dataset is also specified as an output of your `evaluate_model` node. Under `pipeline.py` of your `data_processing` pipeline, specify the `output` of your `evaluate_model` to be the metrics dataset. Note that it is crucial that the output dataset exactly matches the name of the tracking dataset as specified in the config file.
+
+The node of the `evaluate_model` on the pipeline should look like this:
+
+```python
+node(
+    func=evaluate_model,
+    inputs=["regressor", "X_test", "y_test"],
+    name="evaluate_model_node",
+    outputs="metrics"
+)
+```
+
+Let's repeat the same steps for setting up the `companies_column` dataset. For this dataset we are trying to log the column that contains the list of companies as outlined in `companies.csv` under `/data/01_raw`. Let's set up the tracked dataset from the `preprocess_companies` function under the `data_processing` pipeline by returning the data under a key value pair, as shown below:
+
+```python
+def preprocess_companies(companies: pd.DataFrame) -> pd.DataFrame:
+    """Preprocesses the data for companies.
+
+    Args:
+        companies: Raw data.
+    Returns:
+        Preprocessed data, with `company_rating` converted to a float and
+        `iata_approved` converted to boolean.
+    """
+    companies["iata_approved"] = _is_true(companies["iata_approved"])
+    companies["company_rating"] = _parse_percentage(companies["company_rating"])
+    return companies, {"columns": companies.columns.tolist(), "data_type": "companies"}
+```
+
+Again, we will need to ensure that the dataset is also specified as an output on `pipeline.py` under the `data_processing` pipeline, as follows:
+
+```python
+node(
+    func=preprocess_companies,
+    inputs="companies",
+    outputs=["preprocessed_companies", "companies_columns"],
+    name="preprocess_companies_node",
+)
+```
+
+Having set up both datasets, we are now ready to generate our first set of experiment tracking data!
+
+## Generating Run data
+
+One of the beauty of native experiment tracking in Kedro is that all tracked data are generated and stored each time you do a Kedro run. Hence, to generat the data, simply do:
+
+```bash
+Kedro run
+```
+
+After the run completes, under `data/09_tracking`, you will now see 2 folders, `companies_column.json` and `metrics.json`. On performing a Kedro run after setting up the tracking datasets, Kedro will generate a folder under the dataset name for each tracked dataset, with each folder containing another folder named under the timestamp of the run. Each subsequent Kedro run will generate a new folder named after the timestamp of the run, in the same directory of the tracked dataset.
+
+You will also now see the `session_store.db` generated from your first Kedro run after enabling experiment tracking, which is used to store all the generated run metadata, alongside the tracking dataset, to be exposed via the GraphQL endpoint for Kedro-Viz.
```

Comment:
```suggestion
You will also see the `session_store.db` generated from your first Kedro run after enabling experiment tracking, which is used to store all the generated run metadata, alongside the tracking dataset, to be exposed via the GraphQL endpoint for Kedro-Viz.
```

Reply:
Do we have any docs explaining how a user can access/use the `GraphQL` endpoint? Do we even want them to directly use this endpoint? If not, I would just remove the mention of GraphQL here.

Reply:
The usage of the graphql endpoint relates to the Kedro-viz repo and is documented in our latest architecture docs

Reply:
In that case I would link it here. But I still wonder if it's actually relevant to mention it here, or whether it's a bit of information overload.

Reply:
indeed, I'll take out the mention of graphql